astrophot/image/window_object.py

```diff
@@ -576,98 +610,98 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self == other
 
-    @torch.no_grad()
-    def __gt__(self, other):
-        results = list((sw > ow).view(-1) for sw, ow in zip(self, other))
-        return torch.all(torch.cat(results))
-
-    @torch.no_grad()
-    def __ge__(self, other):
-        results = list((sw >= ow).view(-1) for sw, ow in zip(self, other))
-        return torch.all(torch.cat(results))
-
-    @torch.no_grad()
-    def __lt__(self, other):
-        results = list((sw < ow).view(-1) for sw, ow in zip(self, other))
-        return torch.all(torch.cat(results))
-
-    @torch.no_grad()
-    def __le__(self, other):
-        results = list((sw <= ow).view(-1) for sw, ow in zip(self, other))
-        return torch.all(torch.cat(results))
-
-    # Window adjustment operators
-    @torch.no_grad()
-    def __add__(self, other):
-        try:
-            new_windows = list(sw + ow for sw, ow in zip(self, other))
-        except TypeError:
-            new_windows = list(sw + other for sw in self)
-        return self.__class__(window_list=new_windows)
-
-    @torch.no_grad()
-    def __sub__(self, other):
-        try:
-            new_windows = list(sw - ow for sw, ow in zip(self, other))
-        except TypeError:
-            new_windows = list(sw - other for sw in self)
-        return self.__class__(window_list=new_windows)
-
-    @torch.no_grad()
-    def __mul__(self, other):
-        try:
-            new_windows = list(sw * ow for sw, ow in zip(self, other))
-        except TypeError:
-            new_windows = list(sw * other for sw in self)
-        return self.__class__(window_list=new_windows)
-
-    @torch.no_grad()
-    def __truediv__(self, other):
-        try:
-            new_windows = list(sw / ow for sw, ow in zip(self, other))
-        except TypeError:
-            new_windows = list(sw / other for sw in self)
-        return self.__class__(window_list=new_windows)
-
-    @torch.no_grad()
-    def __iadd__(self, other):
-        try:
-            for sw, ow in zip(self, other):
-                sw += ow
-        except TypeError:
-            for sw in self:
-                sw += other
-        return self
-
-    @torch.no_grad()
-    def __isub__(self, other):
-        try:
-            for sw, ow in zip(self, other):
-                sw -= ow
-        except TypeError:
-            for sw in self:
-                sw -= other
-        return self
-
-    @torch.no_grad()
-    def __imul__(self, other):
-        try:
-            for sw, ow in zip(self, other):
-                sw *= ow
-        except TypeError:
-            for sw in self:
-                sw *= other
-        return self
-
-    @torch.no_grad()
-    def __itruediv__(self, other):
-        try:
-            for sw, ow in zip(self, other):
-                sw /= ow
-        except TypeError:
-            for sw in self:
-                sw /= other
-        return self
+    # @torch.no_grad()
```

Comment:
Do you want to keep this section of code for future reference?

Perhaps add a note to the top of the block to future developer about why they may want to look at these lines and reactivate some of the code.  Your future self may be the thankful beneficiary.

Reply:
I think I need to review these more closely. With the new meaning that `Window` has taken on, some of these aren't as well defined as they were before. Others may still be useful

Reply:
I've decided to remove them. I don't think they quite make sense in the new more substantial role that windows have taken on (they may never have made sense). If I need the code I can always go back to an old commit and retrieve it